src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -268,6 +268,7 @@
     <InternalsVisibleToTest Include="Roslyn.VisualStudio.CSharp.UnitTests" />
     <InternalsVisibleToTest Include="Roslyn.VisualStudio.DiagnosticsWindow" />
     <InternalsVisibleToTest Include="Roslyn.VisualStudio.Services.UnitTests" />
+    <InternalsVisibleToTest Include="Roslyn.VisualStudio.Test.Utilities" />
```

Comment:
Removed. Microsoft.VisualStudio.LanguageServices.Guids shouldn't be changing, so the constants were copied inline.